commit a5cb55a919dd1b07a4c3bb3a1e5dbbe7b361b7f4 (refs/changes/80/3380/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-15T18:31:47-08:00 (1 year, 8 months ago)
    
    TRITON-139 Triton's config-agents should poll at 120s interval as a stopgap (update default pollInterval)
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/agent.js b/agent.js
index 0996195..25736f8 100644
--- a/agent.js
+++ b/agent.js
@@ -68,7 +68,7 @@ if (configPath) {
 	config = {
 		instances: [],
 		logLevel: 'info',
-		pollInterval: 60000
+		pollInterval: 120000
 	};
 }
 
